Day5/style.css

```diff
@@ -0,0 +1,13 @@
+#color-field {
+  width: 200px;
+  height: 200px;
+  background-color: rgb(0,0,0);
+  border-radius: 50%;
+  margin: auto;
+  margin-top: 2%;
+}
+
+#clear-butt {
+  width: 20%;
+  height: 20%;
```

Comment:
Do not use `%` for css unit as it will break when you resize the screen. Try resizing the screen and you will see the result.

The button is working fine on it's own, because it will make sure the content fits with it's size.

If you just want to increase the button size, try add `padding`. For example if you want to make your button goes wider, use:
```
#clear-butt {
  padding-left: 60px;
  padding-right: 60px;
}
```